app/models/submitChangeDestination/TraderModel.scala

```diff
@@ -186,7 +187,10 @@ object TraderModel extends ModelConstructorHelpers {
       traderExciseNumber = None,
       traderName = Some(mandatoryPage(FirstTransporterNamePage)),
       address = Some(AddressModel.fromUserAddress(mandatoryPage(FirstTransporterAddressPage))),
-      vatNumber = Some(mandatoryPage(FirstTransporterVatPage)),
+      vatNumber = request.userAnswers.get(FirstTransporterVatPage) match {
```

Comment:
I _think _ the `request.userAnswers.get` would already retrieve the value from the IE801 if it doesn't exist in UserAnswers?